include/psa/crypto_values.h

```diff
@@ -1153,8 +1153,7 @@
  * \return              The corresponding AEAD algorithm with the specified
  *                      length.
  * \return              Unspecified if \p alg is not a supported
- *                      AEAD algorithm or if \p tag_length is not valid
- *                      for the specified AEAD algorithm.
+ *                      AEAD algorithm.
```

Comment:
The return value is unspecified if `tag_length` is not valid. Please undo the latest change here.

Reply:
This was requested by @athoelke (and pointed out by @ronald-cron-arm that I forgot to do the change) here: https://github.com/ARMmbed/mbedtls/pull/4067#discussion_r580214656